highlight/highlight/langDefs/chpl.lang

```diff
@@ -15,26 +15,26 @@ Keywords={
          "begin", "borrowed", "break", "by",
          "catch", "class", "cobegin", "coforall",
          "config", "const", "continue",
-         "delete", "dmapped", "do", "domain",
+         "delete", "dmapped", "do",
          "else", "enum", "except", "export", "extern",
          "false", "for", "forall",
-         "if", "in", "index", "inline", "inout", "iter",
-         "label", "lambda", "let", "local",
+         "if", "in", "index", "init", "init=", "inline", "inout", "iter",
```

Comment:
Ditto other PR.  I don't think `init` and `init=` should be highlighted, but I think the other changes are right

Reply:
And again, sorry for the delay and thanks for the contribution!

Reply:
@lydia-duncan I've updated it